### PR TITLE
ErrorPageServlet cleanup compiler warnings

### PR DESCRIPTION
**Pull request by bradbeck**
Cleaned up the code around `errorName` which was never getting set.

### CODE REVIEW

File: components/nexus-core/src/main/java/org/sonatype/nexus/internal/web/ErrorPageServlet.java [hunk 1]
```diff
@@ -98,19 +98,23 @@ public ErrorPageServlet(final Provider<SystemStatus> systemStatus,
...
     String servletName = (String) request.getAttribute(ERROR_SERVLET_NAME);
+    @SuppressWarnings("unused")
     String requestUri = (String) request.getAttribute(ERROR_REQUEST_URI);
     Integer errorCode = (Integer) request.getAttribute(ERROR_STATUS_CODE);
     String errorMessage = (String) request.getAttribute(ERROR_MESSAGE);
-    Class causeType = (Class) request.getAttribute(ERROR_EXCEPTION_TYPE);
+    @SuppressWarnings("unused")
+    Class<?> causeType = (Class<?>) request.getAttribute(ERROR_EXCEPTION_TYPE);
+    @SuppressWarnings("unused")
     Throwable cause = (Throwable) request.getAttribute(ERROR_EXCEPTION);
```
Comment (jdillon) [inline components/nexus-core/src/main/java/org/sonatype/nexus/internal/web/ErrorPageServlet.java]: you can probably just comment these out for now, we may need to revisit using these details for future refinement, but they aren't presently used, so commenting them out vs. "unused" warning suppression here may be better in the short-term.

Comment (bradbeck) [inline components/nexus-core/src/main/java/org/sonatype/nexus/internal/web/ErrorPageServlet.java]: Commenting out those lines causes the "unused" warnings to migrate to the now unused constant string. 6 of one, half-dozen of another...

**Comment by bradbeck** [comment]
http://bamboo.s/browse/NX3-OSSF460-1

**Comment by jdillon** [comment]
+1

**Comment by mcculls** [comment]
+1

**Pull request merged**